release/ray_release/buildkite/step.py

```diff
@@ -64,18 +66,18 @@ def get_step(
 
     step = copy.deepcopy(DEFAULT_STEP_TEMPLATE)
 
-    cmd = f"./release/run_release_test.sh \"{test['name']}\" "
+    cmd = ["./release/run_release_test.sh", test["name"]]
 
     if report and not bool(int(os.environ.get("NO_REPORT_OVERRIDE", "0"))):
-        cmd += " --report"
+        cmd += ["--report"]
 
     if smoke_test:
-        cmd += " --smoke-test"
+        cmd += ["--smoke-test"]
 
     if ray_wheels:
-        cmd += f" --ray-wheels {ray_wheels}"
+        cmd += ["--ray-wheels", ray_wheels]
 
-    step["command"] = cmd
+    step["plugins"][0][DOCKER_PLUGIN_KEY]["command"] = cmd
```

Comment:
Why do we need to overwrite the plugin command? Looking at the plugin docs we should just continue to use the regular `step["command"]`? https://github.com/buildkite-plugins/docker-buildkite-plugin